imhotep/tools.py

```diff
@@ -40,6 +40,14 @@ def invoke(self, dirname, filenames=set(), linter_configs=set()):
         """
         retval = defaultdict(lambda: defaultdict(list))
         if len(filenames):
+            extensions = [e[1:] for e in self.get_file_extensions()]
```

Comment:
Addressed in https://github.com/justinabrahms/imhotep/pull/91/commits/636bb8ea729da2a3edf849f1d81cff0fea5b5833